shared/constants/provision.js

```diff
@@ -8,7 +8,7 @@ import HiddenString from '../util/hidden-string'
 export const waitingKey = 'provision:waiting'
 
 // Do NOT change this. This is the value used by the daemon also so this way we can ignore it when they do it / when we do
-export const cancelDesc = 'kex canceled by caller'
+export const cancelDesc = 'Input canceled'
```

Comment:
confirm that the comment's still correct and the daemon changed their version?

Reply:
yes, works on pass and paperkey

Reply:
we still have an error that has that string in it, but i'm not sure what `cancelDesc` is for.

Reply:
kex2/base.go:var ErrCanceled = errors.New("kex canceled by caller")

Reply:
in my testing, we send this to cancel the login process when clicking back. it seems to function correctly on every step so i think we're good